### PR TITLE
add InsureDAO

### PR DESCRIPTION
**Pull request by hirorylabo**
##### Twitter Link:
https://twitter.com/insuredao

##### List of audit links if any:

Peckshield
https://drive.google.com/file/d/1RTwAYuPBmQCVrmor-iZ9dFAGlCkdMtdR/view?usp=sharing

##### Website Link:
https://www.insuredao.fi/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):

https://app.insuredao.fi/img/logo.svg

##### Current TVL:
------ TVL ------
ethereum                  136.68 k
ethereum-pool2            130.22 k
ethereum-staking          140.23 k

total                    136.68 k 

##### Chain:
Ethereum

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)

https://www.coingecko.com/en/coins/insuredao

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)

https://coinmarketcap.com/currencies/insuredao/

##### Short Description (to be shown on DefiLlama):
What makes InsureDAO different is our ability to handle everything from creating and selling insurance, to managing surplus funds. We enable DeFi protocols to easily provide full-featured insurance services to users, whilst ensuring maximum returns for our underwriters.

##### Token address and ticker if any:

$INSURE
https://etherscan.io/token/0xd83ae04c9ed29d6d3e6bf720c71bc7beb424393e

##### Category (full list at https://defillama.com/categories) *Please choose only one:

Insurance  (https://defillama.com/protocols/Insurance)

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
None

##### forkedFrom (Does your project originate from another project):
None

##### methodology (what is being counted as tvl, how is tvl being calculated):
Underwrite Assset Liquidty is TVL https://app.insuredao.fi/underwrite
Single Staking of $INSURE(it's called "veINSURE","vlINSURE") is Staking TVL
Uniswap v2 Liquidty Farming of $INSURE is Pool2 TVL

### CODE REVIEW

File: projects/insurace/index.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ async function eth(timestamp, ethBlock) {
         }
         balances[address] = BigNumber(balances[address] || 0).plus(balance).toFixed();
     })
+    console.log(balances)
```
Comment (g1nt0ki) [inline projects/insurace/index.js:43]: hi, can you remove the console log from here and other places

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
can you delete the `insuredao_old` folder

**Comment by hirorylabo** [comment]
Thank you for your review.
I fixed them.

**Comment by llamatester** [comment]
The adapter at projects/insurace exports TVL: 
        
 
 
    avalanche                 4.61 M
    polygon                   4.66 M
    bsc                       2.98 M
    ethereum                  6.58 M
    
    total                    18.83 M

**Comment by llamatester** [comment]
The adapter at projects/insuredao exports TVL: 
        
 
 
    ethereum                  136.28 k
    ethereum-staking          136.56 k
    ethereum-pool2            127.48 k
    staking                   136.56 k
    pool2                     127.48 k
    
    total                    136.28 k

**Comment by llamatester** [comment]
The adapter at projects/insurace exports TVL: 
        
 
 
    polygon                   4.66 M
    avalanche                 4.61 M
    bsc                       2.98 M
    ethereum                  6.58 M
    
    total                    18.83 M

**Comment by llamatester** [comment]
The adapter at projects/insuredao exports TVL: 
        
 
 
    ethereum                  136.28 k
    ethereum-staking          136.56 k
    ethereum-pool2            127.48 k
    staking                   136.56 k
    pool2                     127.48 k
    
    total                    136.28 k

**Pull request merged**